modules/exploits/unix/webapp/trixbox_ce_auth_rce.rb

```diff
@@ -32,7 +32,7 @@ def initialize(info = {})
         },
       'Targets'        =>
         [
-          [ 'Trixbox CE <= v2.8.0.4', { }]
+          [ 'Trixbox CE v2.8.0.4', { }]
```

Comment:
If this affects older versions and you have confirmed this, please revert this change. Only the title of the exploit module needed to be changed.

Reply:
Again this issue was confirmed to affect versions prior to 2.8.0.4 according to the contributor. Therefore this is okay.